frontend/src/Constants/CreateAssetConstants.js

```diff
@@ -0,0 +1,4 @@
+export const STATE_AVAILABLE_ENUM = "0";
+export const STATE_NOT_AVAILABLE_ENUM = "1";
+export const STATE_WAITING_FOR_RECYCLING_ENUM = "2";
+export const STATE_RECYCLED_ENUM = "3";
```

Comment:
Fix these enum match witch BE.
    Available=0
    NotAvailable=1
    Assigned=2
    WaitingForRecycling=3
    Recycled=4